README.md

```diff
@@ -54,248 +99,292 @@ $(SRCROOT)/Carthage/Build/iOS/CryptoSwift.framework
 $(SRCROOT)/Carthage/Build/iOS/EthereumKit.framework
 $(SRCROOT)/Carthage/Build/iOS/FMDB.framework
 $(SRCROOT)/Carthage/Build/iOS/SipHash.framework
-$(SRCROOT)/Carthage/Build/iOS/TrustKit.framework
 $(SRCROOT)/Carthage/Build/iOS/OstWalletSdk.framework
 ```
 
-- Create `OstWalletSdk.plist` file. Following is the default configurations.
+
+<br>
+
+![copy-framework-file](https://dev.ost.com/platform/docs/sdk/assets/add-dependency-framework-files.png)
+
+
+
+#### v). Adding SDK configuration file
+
+Create `OstWalletSdk.plist` file. This file has configuration attributes used by OstWalletSdk. You should copy paste the configuration values from below snippet.
+
+
+Copy paste this configuration file.
 
 ```
 <?xml version="1.0" encoding="UTF-8"?>
  <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
  <plist version="1.0">
  <dict>
- <key>BlockGenerationTime</key>
- <integer>3</integer>
- <key>PricePointCurrencySymbol</key>
- <string>USD</string>
- <key>RequestTimeoutDuration</key>
- <integer>30</integer>
- <key>PinMaxRetryCount</key>
- <integer>3</integer>
- <key>SessionBufferTime</key>
- <integer>3600</integer>
- <key>UseSeedPassword</key>
- <false/>
+    <key>BlockGenerationTime</key>
+    <integer>3</integer>
+    <key>PricePointTokenSymbol</key>
+    <string>OST</string>
+    <key>PricePointCurrencySymbol</key>
+    <string>USD</string>
+    <key>RequestTimeoutDuration</key>
+    <integer>30</integer>
+    <key>PinMaxRetryCount</key>
+    <integer>3</integer>
+    <key>SessionBufferTime</key>
+    <integer>3600</integer>
+    <key>UseSeedPassword</key>
+	<false/>
  </dict>
  </plist>
- ```
-- _BlockGenerationTime_: The time in seconds it takes to mine a block on auxiliary chain.
-- _PricePointCurrencySymbol_: It is the symbol of quote currency used in price conversion.
-- _RequestTimeoutDuration_: Request timeout in seconds for https calls made by ostWalletSdk.
-- _PinMaxRetryCount_: Maximum retry count to get the wallet Pin from user.
-- _SessionBufferTime_: Buffer expiration time for session keys in seconds.
-- _UseSeedPassword_: Uses mnemonics and password to generate seed.
+```
+
+1. BlockGenerationTime: The time in seconds it takes to mine a block on auxiliary chain.
+2. PricePointTokenSymbol: This is the symbol of base currency. So its value will be `OST`.
+3. PricePointCurrencySymbol: It is the symbol of quote currency used in price conversion.
+4. RequestTimeoutDuration: Request timeout in seconds for https calls made by ostWalletSdk.
+5. PinMaxRetryCount: Maximum retry count to get the wallet Pin from user.
+6. SessionBufferTime: Buffer expiration time for session keys in seconds.
+7. UseSeedPassword: Uses mnemonics and password to generate seed.
```

Comment:
The seed password is actually salt to `PBKDF2` used to generate seed from the mnemonic.

More info available here:
https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki#from-mnemonic-to-seed

When `UseSeedPassword ` set to true, different deterministic salts are used for different keys.